Makefile

```diff
@@ -351,6 +351,7 @@ component-test: export COMPONENT_TEST := true
 component-test: export BABEL_ENV := test
 component-test: ##@test Run tests once in NodeJS
 	# Here we create the gyp bindings for nodejs
+	yarn install && \
```

Comment:
@J-Son89 have to add this back
we can always skip `yarn install` manually

Reply:
yeah, that's fine. I just removed while not in CI. but knew this day would come. :)

and yes we can skip manually 👍